### PR TITLE
docs: Final update for UG

### LINKED ISSUE #109: [PE-D][Tester B] "AddressBook" not cleaned up in the UserGuide
There are traces of the old addressbook user guide which the current one is adapted from. It might potentially confuse non-tech users.

For instance:

Suggestion:
Perhaps you could consider rename all the "address book" phrases in the User Guide to something that is more relevant to the current product.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: cyli133/ped#2

### PR DESCRIPTION
**Pull request by Infrix**
## Changes
* [Docs] Update UG with additional tutorial and details
* [Docs] Bug fixes from PE Dry run
* [Docs] Update UG with suggestions from consultation

* Closes 
#109 
#125 
#126 
#139

## Note
* _This should be the final draft for the UG. Do nitpick about anything out of place :)_

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -225,7 +228,71 @@ If you accidentally cancel an order, you can still revert its status!
...
+</div>
+
+Before we start, head over to [_Add your first customer and order_](#add-your-first-customer-and-order) to include _Lyndon Edwards_ and the _Banana Cake_ order.
+However, do not clear the application data as we will be using the customer for this tutorial.
+
+Rewarding your customers points is an effortless process on LoyaltyLift using 
+[`setpoints`](#setting-reward-points-for-a-customer--setpoints) or 
+[`addpoints`](#addingsubtracting-points-for-a-customer--addpoints).
+
+You would like to add points to _Lyndon Edwards_ for his _Banana Cake_ order.
```
Comment (CloudHill) [inline docs/UserGuide.md]: Saying "you would like to add points..." feels kind of imposing.

Maybe something like "Let's add points..." or "Let's see how you can add points...", or "In this scenario, we would like to add points..." would be more suitable?

There is also a couple more "you would like to" on lines 266 and 267, but the usage for "Perhaps, you would like to..." on line 267 sounds fine as you started with a "Perhaps" so maybe you could do that too

File: docs/UserGuide.md [hunk 2]
```diff
@@ -247,16 +314,16 @@ Here are some recommended features to explore after this tutorial.
 
 **:information_source: Notes about the command format:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+* Words in `UPPER_CASE` are the parameters to be supplied by you.<br>
+  e.g. `addc n/NAME ...`, `NAME` is a parameter which can be used as `addc n/John Doe ...`.
```
Comment (CloudHill) [inline docs/UserGuide.md:319]: Maybe "that you can supply" instead of "to be supplied by you" would sound better?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -247,16 +314,16 @@ Here are some recommended features to explore after this tutorial.
...
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [p/PHONE_NUMBER]` can be used as `n/John Doe p/12341234` or as `n/John Doe`.
+  e.g. `...n/NAME [q/QUANTITY]` can be used as `...n/Cake q/2` or as `...n/Cake`.
 
 * Items in curly brackets are an indication to use one of the given options, separated by the character `|`.
   e.g. `ct/{ind|ent}` is a parameter that should be used as `ct/ind` or `ct/ent` exactly.
 
-* Parameters can be in any order.<br>
+* Parameters can be in any order by you.<br>
```
Comment (CloudHill) [inline docs/UserGuide.md]: This also sounds a little off, maybe you can say "You can enter parameters in any order" instead

File: docs/UserGuide.md [hunk 4]
```diff
@@ -1033,16 +1114,23 @@ Customer and order data are saved in the hard disk automatically after any comma
 
 All data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about file name:**<br>
+
+* If you are confused why the file name is addressbook.json, this is because LoyaltyLift was built on top of an existing application called addressbook.
+</div>
```
Comment (CloudHill) [inline docs/UserGuide.md]: Are we not changing the filename to `loyaltylift.json`?

**Review by CloudHill — CHANGES_REQUESTED** [review]
Looks good overall! Just some nits here and there (except for the filename bit)

File: docs/UserGuide.md (hunk 1 above)
Comment (Infrix) [reply docs/UserGuide.md]: that is true LOL, i didnt catch that while trying to format the tutorial oops

File: docs/UserGuide.md (hunk 3 above)
Comment (Infrix) [reply docs/UserGuide.md]: wait idk why i added by you just like that HAHAHAH thanks for the catch

File: docs/UserGuide.md [hunk 5]
```diff
@@ -517,6 +590,9 @@ unmarkc CINDEX
 
 Sets a customer's note.
 
+Do you have additional information to remember about your customer? 
+This command helps you to remember anything you want about your customer.
```
Comment (CloudHill) [inline docs/UserGuide.md]: This sounds a little repetitive, maybe you can say something along the lines of "Note that information down with the `setnotec` command!" or "This command lets you note down that information!" for the second line

File: docs/UserGuide.md (hunk 4 above)
Comment (Infrix) [reply docs/UserGuide.md]: I was initially deciding to leave it as addressbook but i think now i managed to find the spot to change to loyaltylift.json

**Review by CloudHill — APPROVED** [review]
Ok, LGTM! Also we haven't actually changed the filename yet but I'll do that real quick

**Pull request merged**